# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/test/download/worker.js
# Commit: abcbfdf1a1ef
# Full Hash: abcbfdf1a1efad1d852db7889ebda328c4e8b73d
# Author: Toshihito Kikuchi <tkikuchi@mozilla.com>
# Date: 2019-11-15 09:53:19
# Regressor Bug: 1567614
# File Overlap Count: 1
# Description:
#   Bug 1588975 - Replace ShellExecuteExW with mozilla::ShellExecuteByExplorer. r=aklotz,asuth
#   
#   The launcher process turns on the `PreferSystem32Images` mitigation policy for
#   the browser process.  Since the mitigation policy is inherited, a process launched
#   by the browser process also has `PreferSystem32Images`.  If an application which
# ==============================================================================

diff -r 36bef03d2b69 -r abcbfdf1a1ef dom/serviceworkers/test/download/worker.js
--- a/dom/serviceworkers/test/download/worker.js	Thu Nov 14 18:36:28 2019 +0000
+++ b/dom/serviceworkers/test/download/worker.js	Thu Nov 14 19:47:29 2019 +0000
@@ -23,6 +23,8 @@
       return new Response("service worker generated download", {
         headers: {
           "Content-Disposition": 'attachment; filename="fake_download.bin"',
+          // Prevent the default text editor from being launched
+          "Content-Type": "application/octet-stream",
           // fake encoding header that should have no effect
           "Content-Encoding": "gzip",
         },